src/SFA.DAS.LevyTransferMatching.Web/Controllers/PledgesController.cs

```diff
@@ -101,5 +101,12 @@ public async Task<IActionResult> Level(LevelPostRequest request)
             await _orchestrator.UpdateCacheItem(request);
             return RedirectToAction("Create", new CreateRequest() { EncodedAccountId = request.EncodedAccountId, CacheKey = request.CacheKey });
         }
-   }
+
+        [HttpGet]
+        [Route("{pledgeId}/confirmation")]
+        public async Task<IActionResult> Confirmation(string pledgeId, string encodedAccountId)
+        {
+            return View(new ConfirmationViewModel() { EncodedAccountId = encodedAccountId, PledgeId = pledgeId });
```

Comment:
I think we should create the viewmodel in a method in the orchestrator (even though it's incredibly simple, it'd follow the pattern).

Reply:
'm not convinced about this one, we'd effectively be calling another method just to create the ViewModel?